config/routes.rb

```diff
@@ -2,6 +2,7 @@
   # get 'users/create'
 
   resources :trips
+  get ':user_id/trips', to: 'trips#get_user_trips'
```

Comment:
@pkrawat1  yes it is like this, should i change it to localhost:3000/user/1/trips